### PR TITLE
imperium fleet ai, recruitment, ship lost event, orks/nids not attacking fixes & reverted #27

### PR DESCRIPTION
**Pull request by Kirzral**
-Fixed a crash when player becomes renegade and game checked what to do when imperium fleet hunts player fleet.

~~-Fixed crash when necron tomb awakens.~~ <---  #17 solved the issue

-Fixed crash when losing a ship to the warp.

-Refactored some vars at obj_en_fleet\Alarm_1 and scr_enemy_ai_e

-Fixed a small bug on recruitment that led to a crash.

-Reverted #27 changes

-Fixed a typo at scr_en_weapon which led to, at least, orks and nids not getting equipped properly. 

Changes have been tested on my end.

### CODE REVIEW

**Comment by OH296** [comment]
@Kirzral your method for solving the problem necron star is out of date planet features are now enumed and are all structures Necron tombs are therefore all structures with characteristics of either sealed or unsealed meaning the finding a necron tomb planet now needs the syntax of : 
scr_get_planet_with_feature(<star>r, P_features.Necron_Tomb);

and testing if the star has an awake tomb uses the function awake_necron_Star(<star>)

All planet feature scripting and functions is held in scr_planetary_feature script

This is my fault thought i'd updated the whole program correctly but some things must have slipped through.

**Comment by Kirzral** [comment]
@OH296 I believe the problem was I was working with the pre-"planet features update" code. I've tested the event again with the changes from main and it's working, so everything's good. I'll take your explanation as reference if I ever need to tinker with planet features again, so thanks for posting it!

If i find anything weird i'll try to push it into this PR/branch/thing (im still figuring how Git, GitHub and the code/language works), since the general idea of it was to have it as a "stuff behaves weirdly or crashes, stuff gets fixed" instead of making a PR for every thing I find, so feel free to push any fix into it if you feel like it.

**Comment by OH296** [comment]
@Kirzral  not a problem, just trying to keep tabs on anything in my knowledge area to help speed things up.

File: objects/obj_en_fleet/Alarm_1.gml [hunk 1]
```diff
@@ -508,7 +508,7 @@ if (obj_controller.faction_status[2]="War") and (action="") and (trade_goods="")
                 pnear=instance_nearest(fnear.x,fnear.y,obj_star);
                 
                 if (instance_exists(pnear)) and (instance_exists(orbiting)){
-                    if (fdis<=500) and (pnear.orbiting!=orbiting){// Case 1; really close, wait for them to make the move
+					if (fdis<=500) and (pnear!=orbiting){// Case 1; really close, wait for them to make the move
```
Comment (GalacticChimp) [inline objects/obj_en_fleet/Alarm_1.gml:511]: can fdis and pnear have better names?

**Comment by GalacticChimp** [comment]
Looks fine, please just rename those 2 vars

File: objects/obj_en_fleet/Alarm_1.gml (hunk 1 above)
Comment (Kirzral) [reply objects/obj_en_fleet/Alarm_1.gml]: Done. Renamed other 3 vars on the file while I was on it.

File: scripts/scr_enemy_ai_e/scr_enemy_ai_e.gml [hunk 2]
```diff
@@ -662,7 +662,7 @@ function scr_enemy_ai_e() {
 	                    onceh=0;
                     
                     
-	                    obj_controller.recruit_corruption[t]=corr;
+	                    obj_controller.recruit_corruption[thiss]=corr;
```
Comment (GalacticChimp) [inline scripts/scr_enemy_ai_e/scr_enemy_ai_e.gml:665]: this~~s~~

Comment (Kirzral) [reply scripts/scr_enemy_ai_e/scr_enemy_ai_e.gml]: not exactly the same var name, but done

**Review by GalacticChimp — APPROVED** [review]

**Pull request merged**